pkg/distro/defs/fedora/distro.yaml

```diff
@@ -228,6 +228,15 @@
               - "zezere_ignition_banner.service"
               - "parsec"
               - "dbus-parsec"
+        "f43 uses new greenboot":
+          when:
+            version_equal: "43"
```

Comment:
Should this be `version_equal` or wouldn't `version_greater_or_equal` be better here (assuming we keep using this new greenboot in f44+ as well?)

Reply:
Correct, thanks for pointing that out. updated in the latest commit.